search/serializers.py

```diff
@@ -300,6 +302,27 @@ class LearningResourceSerializer(serializers.ModelSerializer):
 
     offered_by = ESOfferedByField()
     topics = ESTopicsField()
+    minimum_price = serializers.SerializerMethodField()
+
+    def get_minimum_price(self, instance):
+        """
+        Minimum price from all learning resource runs
+        """
+
+        if hasattr(instance, "runs") and instance.runs:
+            prices = [
+                run.prices.values_list("price", flat=True)
+                for run in instance.runs.all()
+            ]
+
+            if prices:
+                prices = reduce(lambda x, y: x | y, prices)
+
+                return float(min(prices, default=0))
```

Comment:
Is it possible to avoid using `float` for prices here? If we return a `Decimal` here I think it will be passed along as a string, which would avoid floating point precision problems. But I'm not sure how that interacts with the `scaled_float` type I suggested above.

Reply:
I was getting `TypeError: Object of type 'Decimal' is not JSON serializable` with an unconverted decimal. I updated the code to explicitly convert to a string here